### PR TITLE
Workflow: Updating Tpetra and MueLu auto-project

### PR DESCRIPTION
**Pull request by csiefer2**
Workflow update

### CODE REVIEW

**Review by jhux2 — DISMISSED** [review]
lgtm

File: .github/workflows/tpetra_muelu_label_to_project.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Auto assign MueLu or Tpetra labelled issues to MueLu or Tpetra Projects
+name: Auto assign MueLu or Tpetra labelled or titled issues to MueLu or Tpetra Projects
```
Comment (jhux2) [inline .github/workflows/tpetra_muelu_label_to_project.yml:1]: btw, should read "labeled".

Comment (csiefer2) [reply .github/workflows/tpetra_muelu_label_to_project.yml]: Yeah, yeah

**Review by jhux2 — APPROVED** [review]
lgtm

**Pull request merged**